lawnchair/src/ch/deletescape/lawnchair/smartspace/weathercom/models/SunV1CurrentConditionsResponse.java

```diff
@@ -0,0 +1,183 @@
+/*
+ *     Copyright (c) 2017-2019 the Lawnchair team
+ *     Copyright (c)  2019 oldosfan (would)
+ *     This file is part of Lawnchair Launcher.
+ *
+ *     Lawnchair Launcher is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ *
+ *     Lawnchair Launcher is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with Lawnchair Launcher.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package ch.deletescape.lawnchair.smartspace.weathercom.models;
+
+import com.google.gson.annotations.SerializedName;
+
+public class SunV1CurrentConditionsResponse extends BaseModel {
+
+    public SuccessMetadataSchema metadata;
+    public ObservationsSchema observation;
+
+    public static class SuccessMetadataSchema extends BaseModel {
+        public String version;
+        public String transaction_id;
+        public String location_id;
+        public String language;
+        public String units;
+        public String areaID;
+
+        public Long expire_time_gmt;
+        public Integer status_code;
+
+        public Double longitude, latitude;
+
+        public boolean success;
+        public ErrorResponse error;
+    }
+
+    public static class ObservationsSchema extends BaseModel {
+        @SerializedName("blunt_phrase")
+        public String bluntPhrase;
+
+        @SerializedName("class")
+        public String propertyClass;
+
+        @SerializedName("clds")
+        public String clds;
+
+        @SerializedName("day_ind")
+        public String dayInd;
+
+        @SerializedName("dewpt")
+        public Integer dewpt;
+
+        @SerializedName("expire_time_gmt")
+        public Integer expireTimeGmt;
+
+        @SerializedName("feels_like")
+        public Integer feelsLike;
+
+        @SerializedName("gust")
+        public Integer gust;
+
+        @SerializedName("heat_index")
+        public Integer heatIndex;
+
+        @SerializedName("icon_extd")
+        public Integer iconExtd;
+
+        @SerializedName("key")
+        public String key;
+
+        @SerializedName("max_temp")
+        public Integer maxTemp;
+
+        @SerializedName("min_temp")
+        public Integer minTemp;
+
+        @SerializedName("obs_id")
+        public String obsId;
+
+        @SerializedName("obs_name")
+        public String obsName;
+
+        @SerializedName("precip_hrly")
+        public Integer precipHrly;
```

Comment:
As you are already using SerializedName here I suggest you to give all variables even more readable names (just expanding abbreviations should generally be enough)

Reply:
It’s actually something I was working on, but I forgot about it a couple of hours ago